pkg/bastion/shell.go

```diff
@@ -2132,7 +2132,16 @@ GLOBAL OPTIONS:
 						if err := myself.CheckRoles([]string{"admin"}); err != nil {
 							return err
 						}
-
+						if err := dbmodels.UserKeysByIdentifiers(db, c.Args()).Find(&dbmodels.UserKey{}).Error; err != nil {
```

Comment:
@jwessel This error can be some internal/server error, maybe we should check is the type of error no found with
errros.Is(), this way we will be more foolproof, also we should wrap the errors with more context

Reply:
I like this idea, I think it can go with the error wrapping stuff by creating typed errors and writing some helpers to differentiate them easily

can you list your similar ideas here -> https://github.com/moul/sshportal/issues/259 please?